corehq/apps/users/templates/users/commcare_user_confirm_account.html

```diff
@@ -0,0 +1,39 @@
+{% extends "hqwebapp/base_page.html" %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% block title %}{% trans "Confirm Account" %}{% endblock %}
+
+{% block header %}
+  <div class="page-header container">
+    <h1>{% blocktrans %}{{ hr_name }} Registration{% endblocktrans %}</h1>
+  </div>
+{% endblock %}
+
+{% block content %}
+  {% if user.is_account_confirmed and user.is_active %}
+    <div>
+      <p>
+        {% blocktrans %}
+          Your account is already confirmed! Go sign in.
```

Comment:
What if the view just redirected already-confirmed users straight to login, with a message like this?